### PR TITLE
refactor: filter panel

### PR DESCRIPTION
**Pull request by lynnshaoyu**
https://uniswaplabs.atlassian.net/browse/WEB-1278?atlOrigin=eyJpIjoiZjFmODQ4Mzc4YThkNGMzZmJmNmUxY2NlZjMyOTIwNTQiLCJwIjoiaiJ9

### CODE REVIEW

File: src/nft/components/profile/view/FilterSidebar.tsx [hunk 1]
```diff
@@ -181,54 +161,47 @@ const CollectionItem = ({
   }
   return (
     <Row
+      maxWidth="full"
+      overflowX={'hidden'}
+      overflowY={'hidden'}
```
Comment (cbachmeier) [inline src/nft/components/profile/view/FilterSidebar.tsx:166]: can remove the `{}` braces

File: src/nft/components/profile/view/FilterSidebar.tsx [hunk 2]
```diff
@@ -181,54 +161,47 @@ const CollectionItem = ({
...
+      fontWeight="normal"
+      className={`${subheadSmall} ${styles.subRowHover}`}
```
Comment (cbachmeier) [inline src/nft/components/profile/view/FilterSidebar.tsx:168]: non-blocking but you can add all these styles to `subRowHover` to remove a little bit of bloat

File: src/nft/components/profile/view/FilterSidebar.tsx [hunk 3]
```diff
@@ -181,54 +161,47 @@ const CollectionItem = ({
...
+        <Box as="img" borderRadius="round" width="20" height="20" src={collection.image} />
+        <Box
+          as="span"
+          whiteSpace="nowrap"
+          textOverflow="ellipsis"
+          overflow="hidden"
+          paddingLeft="12"
+          paddingRight="14"
+          style={{ minHeight: 15 }}
+          maxWidth={'180'}
```
Comment (cbachmeier) [inline src/nft/components/profile/view/FilterSidebar.tsx:194]: can remove `{}` braces

File: src/nft/css/sprinkles.css.ts [hunk 4]
```diff
@@ -81,6 +81,7 @@ const dimensions = {
   '100': '100px',
   '120': '120px',
   '160': '160px',
+  '180': '180px',
```
Comment (cbachmeier) [inline src/nft/css/sprinkles.css.ts:84]: Since this is only used once, can you remove it from sprinkles and write it in as a custom style for width

**Review by cbachmeier — APPROVED** [review]
Looks good. Minor style changes.

File: src/nft/components/profile/view/FilterSidebar.tsx (hunk 1 above)
Comment (lynnshaoyu) [reply src/nft/components/profile/view/FilterSidebar.tsx:166]: ty ty

File: src/nft/components/profile/view/FilterSidebar.tsx (hunk 2 above)
Comment (lynnshaoyu) [reply src/nft/components/profile/view/FilterSidebar.tsx:168]: done ty

File: src/nft/css/sprinkles.css.ts (hunk 4 above)
Comment (lynnshaoyu) [reply src/nft/css/sprinkles.css.ts:84]: roger that

**Pull request merged**